app/models/user.rb

```diff
@@ -1,8 +1,14 @@
 # frozen_string_literal: true
 class User < ActiveRecord::Base
   PENSION_WISE_API_PERMISSION = 'pension_wise_api_user'
+  BOOKING_MANAGER_PERMISSION  = 'booking_manager'
 
   include GDS::SSO::User
 
   serialize :permissions, Array
+
+  has_many :booking_requests,
+           -> { includes(:slots).order(:created_at) },
```

Comment:
can we change this to:

```
-> { order(:created_at).includes(:slots) },
```

doesn't effect the functionality but my mind spent 20min trying to work out why you where ordering slots by created at.. :(